test_scripts/Defects/5_0/1306_ResetGlobalProperties_does_not_reset_HELPPROMPT_and_VRHELPITEMS_to_default_values.lua

```diff
@@ -0,0 +1,138 @@
+--
+-- Delete app_info.dat file.
+-- SDL started. Application is registered and activated on HMI.
+-- Description:
+-- Steps to reproduce:
+-- 1) Send SetGlobalProperties from mobile (helpPrompt, timeoutPrompt, vrHelpTitle, vrHelp). ==>successfully set
+-- 2) Stop smartDeviceLinkCore in terminal (through ctrl+c) and do not exit from mobile app.
+-- 3) Start smartDeviceLinkCore and HMI.
+-- 4) Mobile App is registered and activated (HMI lvl FULL)
+-- 5) Send SetGlobalProperties from mobile (helpPrompt, timeoutPrompt, vrHelpTitle, vrHelp). ==>successfully set
+-- 6) Send ResetGlobalProperties from mobile (helpPrompt, timeoutPrompt, vrHelpTitle, vrHelp)
+-- Expected result:
+-- 1) SDL should send to HMI UI.SetGlobalProperties("vrHelpTitle", "vrHelp") with default values.
+-- 2) SDL should send to HMI TTS.SetGlobalProperties with "helpPrompt" parameter that should be reset to default value
+--    (smartDeviceLink.ini file contains default values for "helpPrompt"
+-- Actual result:
+-- 1) SDL sends to HMI UI.SetGlobalProperties(vrHelpTitle) without vrHelp - not OK
+-- 2) SDL sends to HMI TTS.SetGlobalProperties("helpPrompt":[ ]) - empty array - not OK.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/Defects/commonDefects')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local actions = require("user_modules/sequences/actions")
+
+--[[ Local Variables ]]
+local requestParams = {
+	helpPrompt = {{ text = "Help prompt", type = "TEXT" }},
+	timeoutPrompt = {{	text = "Timeout prompt", type = "TEXT" }},
+	vrHelpTitle = "VR help title",
+	vrHelp = {{	position = 1, text = "VR help item" }}
+}
+
+local UiParams = {
+	vrHelpTitle = requestParams.vrHelpTitle,
+	vrHelp = requestParams.vrHelp,
+}
+
+local TtsParams = {
+	timeoutPrompt = requestParams.timeoutPrompt,
+	helpPrompt = requestParams.helpPrompt
+}
+
+local allParams = {
+	requestParams = requestParams,
+	UiParams = UiParams,
+	TtsParams = TtsParams
+}
+
+--[[ Local Functions ]]
+local function getValueFromIniFile(path_to_ini, parameter_name)
+	local f = assert(io.open(path_to_ini, "r"))
+	local fileContent = f:read("*all")
+	local ParameterValue
+	ParameterValue = string.match(fileContent, parameter_name .. " =.- (.-)\n")
+	f:close()
+	return ParameterValue
+end
+
+local pathToIniFile = commonPreconditions:GetPathToSDL() .. "smartDeviceLink.ini"
+
+local requesUIparams = {
+	vrHelpTitle = getValueFromIniFile(pathToIniFile, "HelpTitle"),
+	vrHelp = {{ position = 1, text = actions.getConfigAppParams(pAppId).appName }}
+}
+
+local paramHP = getValueFromIniFile(pathToIniFile, "HelpPromt")
+local paramTP = getValueFromIniFile(pathToIniFile, "TimeOutPromt")
+
+local requesTTSparams = {
+	helpPrompt = {{ text = paramHP[1], type = "TEXT"}, {text = paramHP[2], type = "TEXT"}},
```

Comment:
paramHP and paramTP can contain not 2 elements, need to get the array and set values for helpPrompt elements in loop.